modules/special-rules/pages/war-engines-in-close-combat.adoc

```diff
@@ -2,40 +2,61 @@
 
 == War Engine assault moves and charges
 
-You may move your War Engine units in the Assault phase, and you may make charge moves with them if you wish. 
+You may move your War Engine units in the Assault phase, and you may make charge moves with them if you wish.
 This does not affect the War Engine unit's ability to shoot in the Shooting phase in any way; it always fires at full effect.
 
 War Engine units have a comparatively high Assault value, as you will see on their data sheets.
 Lesser units can and must 'gang up' on a War Engine if they are to defeat it in close combat.
 
-You can 'gang up' on War Engine units with a number of regular units equal to half of the War Engine unit's initial Damage Capacity value.
-{blank}[TODO: How do War Engine units -- or a mix of War Engine and regular units -- gang up on on other War Engine units?]
+You can place as many units in base contact with the War Engine unit as its initial Damage Capacity value.
+
+[NOTE.e40k]
+====
+In _Epic 40,000_, you could only 'gang up' with half as many units, which meant that smaller War Engine units like Baneblades could only be attacked by two units -- the same as a regular target!
+====
```

Comment:
Given that strictly speaking this weakens war engines to some extent, is there any other reason for this rule change? 

I ask because I imagine that others may not see "we can't 'gang up' any more on small war engines than we can on regular units" as a problem — or not enough of a problem to justify doubling the number of units that can be in contact for all war engines.

Meantime, just a few minor tweaks:

```suggestion
[NOTE.e40k]
====
In _Epic 40,000_, you could only 'gang up' with half as many units as the initial Damage Capacity.
This meant that against small War Engine units such as the Baneblade you could only place two units in base contact -- the same as against a Vehicle or Infantry target!
====
```

Reply:
The previous discussion appeared to be strongly in favour of increasing it: https://www.facebook.com/groups/1004227353047038/posts/2267032236766537/

But I've started the usual polls to see if people disagree...

Reply:
Some further tweaks: You make a good case on Facebook, and I agree — but I think that the particular point you used in this note was one of the weakest ones. While we don't want to go into depth here, I think that it'd be best to allude to the stronger reasons.

Something like this perhaps:

```suggestion
[NOTE.e40k]
.'Ganging up' on War Engine units
====
In _Epic 40,000_, you can only 'gang up' with *half* as many units as the initial Damage Capacity.
This leads to a number of odd effects, not least that it makes war engines *excessively* difficult to overcome in close combat -- so, we reference the full initial Damage Capacity instead.
====
```